contributors/design-proposals/mount-options.md

```diff
@@ -13,9 +13,31 @@ Mount time options that are operationally important and have no security implica
 
 ## Design
 
-We are going to add support for mount options in PVs as a beta feature to begin with.
+### Mount option support in Persistent Volume Objects
+
+Mount options can be specified as a field on PVs. For example:
+
+``` yaml
+ apiVersion: v1
+  kind: PersistentVolume
+  metadata:
+    name: pv0003
+  spec:
+    capacity:
+      storage: 5Gi
+    accessModes:
+      - ReadWriteOnce
+    persistentVolumeReclaimPolicy: Recycle
+    mountOptions: "hard,nolock,nfsvers=3"
+    nfs:
+      path: /tmp
+      server: 172.17.0.2
+```
+
+
+Beta support for mount options introduced via `mount-options` annotation will be supported for near future
```

Comment:
No it was beta - https://kubernetes.io/docs/concepts/storage/persistent-volumes/#mount-options